### ISSUE TITLE
master needs to be tested - newInbox seems to fail

### ISSUE THREAD

**Issue by w2vy** [rchain-community/rgov#181]
Log shows 

{"master-dictionary dictionary unforgeable" : Nil}

@Bill-Keuntje @jimscarver

**Comment by jimscarver** [comment]
i updates rhobot aand localhost to master both are now broke.

yike master does not run on rhobot or localhost . I get a warning when I npm start
 ```
WARNING in ./src/participate.js 40:15-31
"export 'localhostNETWORK' was not found in './MasterURI.localhost.json'
```
and then in the browser it dies
```
Uncaught TypeError: Cannot read property 'MasterURI' of undefined
    at Module../src/participate.js (participate.js:40)
```

i tried starting with --resolveJsonModule  in webpack but that is a typescript option and did not help

**Comment by Bill-Keuntje** [comment]
Should be fixed by #182

**Comment by jimscarver** [comment]
again i am getting {"master-dictionary dictionary unforgeable" : Nil}
